lib/jekyll-redirect-from/redirector.rb

```diff
@@ -79,8 +79,10 @@ def config_github_url(site)
       end
     end
 
-    def config_baseurl(site)
-      site.config.fetch('baseurl', nil)
+    def config_url(site)
+      url = site.config.fetch('url', nil) || ""
+      baseurl = site.config.fetch('baseurl', nil) || ""
+      File.join url, baseurl
```

Comment:
Could both of the fetches be rewritten as `fetch <KEY>, ""` rather than `fetch <KEY>, nil` || ""`?